src/plugins/kibana/public/discover/index.html

```diff
@@ -176,7 +176,12 @@
 
               </header>
 
-              <visualize ng-if="vis && rows.length != 0" vis="vis" es-resp="mergedEsResp" search-source="searchSource"></visualize>
+              <visualize
+                ng-if="vis && rows.length != 0" vis="vis"
+                es-resp="mergedEsResp"
+                export-title="{{opts.savedSearch.title}}"
```

Comment:
I personally think that the export title should always equal the `SavedObject`s title, regardless of what that title is, and I don't think it's the responsibility of the data export logic to determine the default title for the `SavedObject`.

If "New Saved Search/Visualization" isn't a good default title for `SavedObject`s we open another ticket.